### DISCUSSION TITLE
Welcome to uuWAF Discussions!

### DISCUSSION THREAD

**Discussion by Safe3** [Safe3/uusec-waf#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NoziroDrZ** [comment]
Hi, 
I know the default username is "admin", and the default password is "Passw0rd!".
But i have a " Error: Unable to connect to the database server " on web login page after install. 
I have not touched DockerCompose or the conf file in the container volumes
![téléchargement](https://github.com/user-attachments/assets/f12430bc-469b-47fc-ade0-5fc2f95aef56)

**Reply by Safe3** [reply]
It means that the mysql database docker container can't be started, you should run "docker logs wafdb" to see the error detail

**Reply by Safe3** [reply]
> Hi, I know the default username is "admin", and the default password is "Passw0rd!". But i have a " Error: Unable to connect to the database server " on web login page after install. I have not touched DockerCompose or the conf file in the container volumes ![téléchargement](https://private-user-images.githubusercontent.com/204213910/460985115-f12430bc-469b-47fc-ade0-5fc2f95aef56.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTE0MjUzNTMsIm5iZiI6MTc1MTQyNTA1MywicGF0aCI6Ii8yMDQyMTM5MTAvNDYwOTg1MTE1LWYxMjQzMGJjLTQ2OWItNDdmYy1hZGUwLTVmYzJmOTVhZWY1Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcwMlQwMjU3MzNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05Y2NlZDQ4NjMyMTgyNWFhMGY1N2E1NjNhMTllOGI1NWNkMGNhYWQyN2IyMDMyNWNiZmQ4OTgwZjQyMDJhYmIxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.a666V19EPyUO0D2QzKfFA4-xL8EAbK1Gx7kYZMqRYg0)

You can try the latest v7.0.0, may be it can work perfectly!

**Reply by NoziroDrZ** [reply]
The wahdb is started and i have status up 
in docker logs i just have 4 warning : 
[Warning] TIMESTAMP with implicit DEFAULT value is deprecated. Please use --explicit_defaults_for_timestamp server option (see documentation for more details).
[Warning] A deprecated TLS version TLSv1 is enabled. Please use TLSv1.2 or higher.
[Warning] A deprecated TLS version TLSv1.1 is enabled. Please use TLSv1.2 or higher.
[Warning] CA certificate ca.pem is self signed.

nothing blocking I think

**Reply by NoziroDrZ** [reply]
With latest its better but i dont know if i need OTP ? 
![image](https://github.com/user-attachments/assets/aecb57d3-a112-46a3-b9b2-cb2057ed5747)

**Reply by Safe3** [reply]
The default password is "#Passw0rd" for new version
